src/ASM/SplineFields2D.C

```diff
@@ -26,7 +26,9 @@
 SplineFields2D::SplineFields2D (const ASMs2D* patch,
                                 const RealArray& v, char nbasis,
                                 int nnf, const char* name)
-  : Fields(name), basis(patch->getBasis(nbasis)), surf(patch->getSurface())
+    : Fields(name),
+      basis(patch->getBasis(nbasis)),
+      surf(patch->getBasis(ASM::GEOMETRY_BASIS))
```

Comment:
Something happened with the indentation here? compared to SplineField2D above

Reply:
fixed.